static/docs/install/macos.md

```diff
@@ -20,7 +20,7 @@ Get the PKG package (binary) from the big "Download" button on the
 > [virtual environment](https://packaging.python.org/tutorials/installing-packages/#creating-virtual-environments)
 > or
 > [pipx](https://packaging.python.org/guides/installing-stand-alone-command-line-tools/)
-> for Python 3.6+ to keep your system environment safe.
+> (on Python 3.6+) to encapsulate your local environment.
```

Comment:
I'm fine with either, but the meaning is a little bit different. The previous version is stronger in a sense that it answers second _why_ - `you want to encapsulate your local environment to keep your system environment safe`. One of the main reasons to do this.

Reply:
To me the word safe here may imply that if you don't use a virtual env your computer may be corrupted, hacked or something scary. To me, the main reason for virtual Python envs is avoid dependency conflicts. Not sure that qualifies as safety. 🙂

Reply:
Maybe there's a good term in between that will convey this better... How about "keep your global Python environment healthy"?

Reply:
nah, it's fine as you changed it :) let's not overthink this. link and note should be enough. Just wanted to mention since to me it's close to "your computer may be corrupted" - a lot of users are not able untangle that hell pip can create with their system environment.

Reply:
Especially on Mac where there are site-packages in 3 different system dirs, some of which are protected by special MacOS file security 😋